project-chip/connectedhomeip
# Add Android Open Commissioning Callback API

#### Problem
What is being fixed?  Examples:
* Application can't check open commissioning fail.
* Adding Open Commissioning callback API, Android app can check command success or not.

#### Change overview
Add Open Commissioning callback API in Android Platform/

#### Testing
How was this tested? (at least one bullet point required)
* Check Open commissioning (ECM, BCM)
* Check timeout value changed.
